changes.md

```diff
@@ -0,0 +1,6 @@
+# Changes
+
+- Added `drf-yasg` for swagger API view.
+- Fixed `read_only_fields` for `UserNameSerializer` serializer.
+- Added `django-filters` for `Message` filter backend.
+- Added tests for `Message` list API.
```

Comment:
Could you please explain why this file is needed and provide an explanation in the commit description?

Reply:
Was there for the reviewer's convenience. Removed!